### DISCUSSION TITLE
2021.06.17

### DISCUSSION THREAD

**Discussion by choitree** [malaheaven/issue-tracker#102] [General]
[BE]오늘 API 4개 개발하고, 같이 강의들으면서 개념 학습을 했습니다
아침부터 지금까지 불태웠더니 아주 뿌듯하네요~~~🥕🥕🥕

**Comment by 17-sss** [comment]
**FE**
오늘은 FE에서 쓰이는 임시데이터를 정리했습니다!
생각보다 많이는 못했지만!
**_오늘도 행복합니다!_**
